workitem/workitem_repository.go

```diff
@@ -95,6 +97,43 @@ func (r *GormWorkItemRepository) Load(ctx context.Context, ID string) (*app.Work
 	return convertWorkItemModelToApp(wiType, res)
 }
 
+// LoadTopWorkitem returns top most work item of the list. Top most workitem has the Highest order.
+// returns NotFoundError, ConversionError or InternalError
+func (r *GormWorkItemRepository) LoadTopWorkitem(ctx context.Context) (*app.WorkItem, error) {
+	res := WorkItem{}
+	r.db = r.db.Debug()
+	tx := r.db.Order("execution_order desc").Last(&res)
```

Comment:
@DhritiShikhar what about this?

```go
db := r.db.Model(WorkItem{})
query := fmt.Sprintf("execution_order = (SELECT max(execution_order) FROM %[1]s)",
	workitem.WorkItem{}.TableName(),
)
res = WorkItem{}
db = db.Where(query).First(&res)
```

See [here](https://github.com/almighty/almighty-core/blob/7255e80325f5ca9f9a241564bf003a4d2e492a40/workitem/link/type_repository.go#L266) for an example of more complicated queries.